x-pack/legacy/plugins/epm/server/lib/elasticsearch/ingest_pipeline/ingest_pipelines.ts

```diff
@@ -55,14 +55,41 @@ async function installPipeline({
 }): Promise<AssetReference> {
   const buffer = Registry.getAsset(path);
   const parts = Registry.pathParts(path);
+  const extension = getExtension(path);
   const id = path.replace(/\W/g, '_'); // TODO: replace with "real" pipeline id
   const pipeline = buffer.toString('utf8');
 
-  await callCluster('ingest.putPipeline', { id, body: pipeline });
+  const callClusterParams: {
+    method: string;
+    path: string;
+    ignore: number[];
+    body: any;
+    headers?: any;
+  } = {
+    method: 'PUT',
+    path: `/_ingest/pipeline/${id}`,
+    ignore: [404],
+    body: pipeline,
+  };
+  if (extension === 'yml') {
+    callClusterParams.headers = { ['Content-Type']: 'application/yaml' };
+  }
+
+  // This uses the catch-all endpoint 'transport.request' because we have to explicitly
```

Comment:
It's a pity that it is not supported. @delvedor Does the new client support this?